documents/src/pages/elements/interactive-chart.md

```diff
@@ -442,6 +446,12 @@ chart.config = {
     }
   ]
 };
+chart.chart.priceScale('').applyOptions({
```

Comment:
```suggestion
chart.chart.priceScale().applyOptions({
```

Reply:
This will show an error It required empty string as an id.

Reply:
OK